app.js

```diff
@@ -54,10 +55,12 @@ class App {
         const newConfIds = confIds.filter(id => !(id in this.conferenceStates));
         const removedConfIds = Object.keys(this.conferenceStates).filter(id => confIds.indexOf(id) === -1)
         newConfIds.forEach(newConfId => {
-            const dumpId = uuidv4();
+            const clientId = uuidv4();
             const confState = {
-                dumpId,
-                // TODO: use a set?
+                clientId,
+                confID: extractConferenceName(jvbJson, newConfId),
+                meetingUniqueId: newConfId,
```

Comment:
Also a bit confusing that we use both "conf" and "meeting"

Reply:
meetingId has already been introduced (that's the name of the ID generated in prosody)